velox/dwio/dwrf/writer/StatisticsBuilder.cpp

```diff
@@ -93,6 +93,13 @@ void StatisticsBuilder::merge(
     // Merge size
     mergeCount(size_, other.getSize());
   }
+  if (hll_) {
+    if (auto* otherBuilder = dynamic_cast<const StatisticsBuilder*>(&other)) {
```

Comment:
Shall we add VELOX NOT NULL check here? We should expect all other statistic builder has the same type?